stipple.py

```diff
@@ -0,0 +1,33 @@
+# Import the dependencies
+import argparse
+import PIL
+import random
+from PIL import Image
+from PIL import ImageColor
+import math
+import cv2
+
+# construct the argument parse and parse the arguments
+ap = argparse.ArgumentParser()
+ap.add_argument("-i", "--image", required=True, help="path to input image")
+args = vars(ap.parse_args())
+
+
+img = Image.open(args['image'])
+img = img.convert('LA')
```

Comment:
Can you explain this

Reply:
It is converting the image into Greyscale('L') with alpha transparency('A') .Added to determine how a pixel is rendered when blended with another.